ClientRuntimes/CSharp/Microsoft.Rest.ClientRuntime/project.json

```diff
@@ -24,6 +24,9 @@
         "System.Net.Http": "",
         "System.Net.Http.WebRequest": "",
         "System.Xml": ""
+      },
+      "dependencies": {
+        "Newtonsoft.Json": "6.0.8"
```

Comment:
We need to do this for now, unfortunately, but we will upgrade everyone to 7.0.1 after Jan powershell release.